src/OutputManager.js

```diff
@@ -0,0 +1,28 @@
+import { MissionUtils } from '@woowacourse/mission-utils';
+
+class OutputManager {
+  static printProgressTitle() {
+    MissionUtils.Console.print('실행 결과');
+  }
+
+  static printProgress(cars) {
+    cars.forEach(this.printCarPosition);
+    this.printNewLine();
+  }
+
+  static printCarPosition(car) {
+    const progressBar = '-'.repeat(car.getPosition());
+    MissionUtils.Console.print(`${car.getName()} : ${progressBar}`);
+  }
```

Comment:
재요님의 메서드 분리하는 기준이 궁금합니다! 

저는 책임에 따라 메서드를 분리했어요! 만약 제가 해당 코드를 작성한다면 `printProgress`메서드 하나로 구현했을 거 같아요!
`printCarPosition`의 코드가 따로 메서드로 분리해야 될 정도의 책임을 가지나? 에 대해 고민했을 거 같네요!
```suggestion
  static printProgress(cars) {
    cars.forEach((car)=>{
           const progressBar = '-'.repeat(car.getPosition());
           MissionUtils.Console.print(`${car.getName()} : ${progressBar}`);
    });
    this.printNewLine();
  }
```
이렇게 작성하고 보니, 재요님이 작성하신 코드가 가독성 측면에서 더 좋은 거 같네요 ㅋㅋㅋ

Reply:
두 코드 다 일리가 있다고 생각해서, 다른 분들 의견도 궁금하네용